spec/presenters/hyrax/file_set_presenter_spec.rb

```diff
@@ -285,19 +284,25 @@
       end
 
       context 'with a file' do
-        before do
-          Hydra::Works::AddFileToFileSet.call(file_set,
-                                              file_path, :original_file)
+        let(:file_set) do
+          create_for_repository(:file_set,
+                                user: user,
+                                content: file)
         end
 
         context "when the file is not an image" do
-          let(:file_path) { File.open(fixture_path + '/hyrax_generic_stub.txt') }
+          before do
+            ruby_mock = instance_double(RubyTikaApp, to_json: "{}")
+            allow(RubyTikaApp).to receive(:new).and_return(ruby_mock)
+          end
+
+          let(:file) { fixture_file_upload('/hyrax_generic_stub.txt', 'plain/text') }
 
           it { is_expected.to be_nil }
         end
 
         context "when the file is an image" do
-          let(:file_path) { File.open(fixture_path + '/world.png') }
+          let(:file) { fixture_file_upload('/world.png', 'image/png') }
```

Comment:
I'm seeing these errors ```NoMethodError:
        undefined method `fixture_file_upload'```

Reply:
I do not see this.  How are you running it? I'm doing `bundle exec rspec spec/presenters/hyrax/file_set_presenter_spec.rb`